package-lock.json

```diff
@@ -54147,6 +54149,7 @@
     "packages/health-gorilla": {
       "name": "@medplum/health-gorilla",
       "version": "3.2.8",
+      "extraneous": true,
```

Comment:
Besides breaking glass and manually deleting this whole entry from `package-lock.json`, it doesn't seem like it's possible to get rid of the now gone `packages/health-gorilla` entry.

https://stackoverflow.com/questions/78435644/npm-remove-workspace-from-monorepo
https://github.com/npm/cli/issues/5463
https://github.com/npm/cli/pull/5478